src/fides/api/schemas/saas/shared_schemas.py

```diff
@@ -45,6 +45,12 @@ class IdentityParamRef(BaseModel):
     identity: str
 
 
+class DataSetRef(BaseModel):
```

Comment:
Dataset is one word in Fides
```suggestion
class DatasetRef(BaseModel):
```